modules/core/output_modules.php

```diff
@@ -544,6 +543,15 @@ class Hm_Output_js_data extends Hm_Output_Module {
     protected function output() {
         $res = '<script type="text/javascript">'.
             'var globals = {};'.
+            'window.current_lang = ' . json_encode($this->lang) . ';' .
+            'window.translations = ' . json_encode($this->all_trans()) . ';' .
+            'var trans = function(key, lang = window.current_lang) {' .
```

Comment:
Can we continue with the naming convention hm_? E.g. hm_trans, hm_translations, hm_current_lang?

Reply:
Done too.